examples/example_SF_abst_2-simple.jl

```diff
@@ -9,13 +9,13 @@ using Plots, Colors
 using Test
 
 using Dionysos
-using Dionysos.Problem
 const DI = Dionysos
 const UT = DI.Utils
 const DO = DI.Domain
 const ST = DI.System
 const SY = DI.Symbolic
 const CO = DI.Control
+const PR = DI.Problem
```

Comment:
We should do this transformation for all files in the examples folder ... (if we want it to keep working, but it is true that it takes time to do). I propose to add an issue for this: we can keep this as a future job, because some of these files are still in progress.